siem/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM graylog/graylog:5.0
+FROM graylog/graylog:5.0.1
```

Comment:
Do we need to pin to 5.0.1? Leaving this at 5.0 allows minor updates to be applied.

Reply:
Graylog was flooding log file because it uses old ElasticSearch which responses with deprication-warnings) this was an attempt to get rid of this flooding without success. It does not need to be pinned.

Reply:
Okay, let's go ahead and change that back to 5.0.

Reply:
Okay, let's go ahead and change that back to 5.0.